### DISCUSSION TITLE
Error: failed to create new program: No such project

### DISCUSSION THREAD

**Discussion by rlopez0689** [deta/cloud-docs#736] [Help] [answered]
Hi!

I'm new to Deta, and was trying to deploy a FastAPI project. But when doing the `deta new` i get:

```
Error: failed to create new program: No such project
```

I was searching at the discussions for answers, and look that i need a project to be created (via ui) although the default project should exist. The thing is that when i go to `https://web.deta.sh/home` i get an empty page. 

![Screenshot 2023-01-27 at 12 17 48](https://user-images.githubusercontent.com/8969295/215163944-9cda1586-03d7-40e9-b899-3685f9f307ba.png)

And when inspecting the page i see this error: (already try with Chorme and Safari, same result)
```
Uncaught (in promise) TypeError: Cannot read properties of null (reading 'length')
```

Maybe something went wrong on the creation of my account? Is something i can fix or there is someone on Deta that can help me with this?

**Comment by aavshr** [comment, marked as answer]
Hi @rlopez0689 very sorry for this, was an issue on our end, can you please try again now and see if the issue persists?

**Reply by rlopez0689** [reply]
Hi @aavshr. Yes, is working as expected now.

Thanks so much for the quick help.

**Comment by HatGuy68** [comment]
Hi, I was also facing the same issue initially. It was caused due to not creating even a default project. It can be avoided be automatically creating a default project or improving the message to include that you need to create a project from the website. Hope it helps a new user.
